osu.Game/Graphics/UserInterface/FilteringSearchList.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using osu.Framework;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+
+namespace osu.Game.Graphics.UserInterface
+{
+    public class FilteringSearchList<T> : Container where T : Drawable

```

Comment:
also what's the point of making this generic? you never use the generic-ness of it.